CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ Full documentation for rocFFT is available at [rocfft.readthedocs.io](https://ro
 - rocFFT now automatically allocates a work buffer if the plan
   requires one but none is provided.
 
-## [(Unreleased) rocFFT 1.0.9 for ROCm 4.0.0]
+## [rocFFT 1.0.9 for ROCm 4.0.0]
```

Comment:
So, if I remember correctly,  the whole ROCm 4.0.0 is based on ROCm 3.10.0 with few fixes and no new features. For rocFFT, the original planned commits for ROCm 4.0.0 will all the way push to 4.0.1.